.github/workflows/ci-slow.yml

```diff
@@ -260,27 +293,22 @@ jobs:
     secrets: inherit
   custom-ubuntu-integration-test:
     if: github.event.pull_request.draft == false
-    needs: run-slow-ci-label-is-set
+    needs: [run-slow-ci-label-is-set, custom-ubuntu-unit-test]
     strategy:
       matrix:
-        os: [arc-runner-set]
-        python-version: ['3.9', '3.10', '3.11', '3.12']
-        test_environment:
-          - default
-          - docker-server-docker-orchestrator-mysql
-          - docker-server-docker-orchestrator-mariadb
+        os: [ubuntu-latest]
+        python-version: ['3.9', '3.10', '3.12']
+        test_environment: [default, docker-server-docker-orchestrator-mariadb]
         exclude:
           # docker is time-consuming to run, so we only run it on 3.9
-          - test_environment: docker-server-docker-orchestrator-mysql
-            python-version: '3.10'
-          - test_environment: docker-server-docker-orchestrator-mysql
-            python-version: '3.11'
-          - test_environment: docker-server-docker-orchestrator-mysql
+          - test_environment: default
```

Comment:
we where never doing them all because they just take really long time when used in docker, for that we do full test in default and only one in docker-server-docker orchestrator